NetworkLayerTests/FakeModels.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FakeModels.swift
+//  NetworkLayerTests
+//
+//  Created by Pierre on 18/11/2022.
+//
+
+@testable import NetworkLayer
+import Foundation
+
+class FakeModels {
```

Comment:
```suggestion
enum FakeModels {
```
```suggestion
struct FakeModels {
```
Pourquoi une classe ici ? Une enum ou une struct sont plus légères pour le programme.